test/model-inheritance.test.js

```diff
@@ -239,7 +239,8 @@ describe('Model class inheritance', function() {
         );
 
       assert.deepEqual(User.settings, {
-        forceId: true,
+        // forceId is set to 'auto' in memory if idProp.generated && forceId !== false
+        forceId: 'auto',
```

Comment:
This change makes sense to me. We are basically treating "auto" as "forceId was not specified by the user, but it should be enabled because id is auto-generated".